include/bluetooth/mesh/light_ctrl_srv.rst

```diff
@@ -0,0 +1,427 @@
+.. _bt_mesh_light_ctrl_srv_readme:
+
+Light Lightness Control Server
+##############################
+
+The Light Lightness Control (LC) Server controls a single :ref:`bt_mesh_lightness_srv_readme` instance on the same device with a state machine.
+
+The state machine defines common behavior for a Light fixture through three states, each with its own timing parameters and light levels.
+As input to its state machine, the Light LC Server listens for Sensor and OnOff messages.
+
+In addition to the state machine, the Light LC Server may optionally use a regulator to control the ambient illuminance in the room.
+The regulator requires illuminance sensor readings acting as feedback for its regulator loop.
+
+The Lightness Control Server is always disabled by default, and must be enabled by a :ref:`bt_mesh_light_ctrl_cli_readme`.
+
+Overview
```

Comment:
This isn't in the template, but given the size of the model, I've added an Overview. Check placing and introductory sentence, please.